src/pkg/account/account.go

```diff
@@ -88,3 +91,12 @@ func (a Account) ID() string {
 
 	return a.Config[a.Provider.String()+"-tenant-id"]
 }
+
+func (a Account) GetAccountConfigForHash() (map[string]any, error) {
```

Comment:
can you please add a unit test that checks every account type to ensure it provides a hasher?  We want to make sure that on the next account type addition we'll automatically fail the test if it doesn't include a hashing func.